### ISSUE TITLE
Setting up multiple routes within a single actionfile

### ISSUE THREAD

**Issue by D4V3M0NK** [Ylianst/MeshCentral#615]
As mentioned in #588 it would be hugely beneficial for the ability for multiple maps to be set up within a single meshcmd "actionfile" something like [this](https://github.com/Ylianst/MeshCentral/files/3746189/meshactionMultipleRoutes.txt), ie setting up multiples connections, with the idea of only running one instance of meshcmd but two (or more) maps. When supporting our remote servers, we typically need SSH, MySQL, HTTP and HTTPS.

**Comment by Ylianst** [comment]
Agreed. I am traveling a lot in the next 3 weeks, but I want to do this when I get back. Right now, you can run MeshCMD many times to get the same effect, but it's not ideal.

**Comment by D4V3M0NK** [comment]
Not sure if this is an enhancement to this issue, or one in general: but when using MCR you can specify a map's port as "0", in which case the app automatically finds a spare port.  I notice that when you do this under an actionfile, it takes the value literally. Would be great to have this "auto-assign" port if the local port is set to 0, otherwise the literal value is used.  

Would be happy to post this as an issue to meshcmd if it makes more sense there than here?

**Comment by Ylianst** [comment]
Makes sense here. I agree that port 0 should assign a random port. I need to take a day and improve MeshCMD routing in many ways.
